src/index.ts

```diff
@@ -88,6 +94,10 @@ export async function resolveSchema (config: Config): Promise<Schema> {
               })
           }
         })
+     case 'schema':
+       const schemaSource = require(resolve(config.file))
```

Comment:
Not sure if we are talking the same thing. I don't see how`path.join` is breaking the convention. If you think so, provide your scenario and example.